core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -185,6 +183,30 @@ class ConsoleConsumerTest {
 
   }
 
+  @Test(expected = classOf[IllegalArgumentException])
+  def shouldExitOnUnrecognizedNewConsumerOption(): Unit = {
+
+    // Override exit procedure to throw an exception instead of exiting, so we can catch the exit
+    // properly for this test case
+    Exit.setExitProcedure((_, message) => throw new IllegalArgumentException(message.orNull))
```

Comment:
This comment seems redundant given the code.

Reply:
Yeah, I am going to remove it from all the other places in the same test scala class